src/Microsoft.OData.Core/Build.NuGet/Microsoft.OData.Core.Nightly.Release.nuspec

```diff
@@ -12,7 +12,7 @@
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>Classes to serialize, deserialize and validate OData JSON payloads. Supports OData v4.</summary>
     <description>Classes to serialize, deserialize and validate OData JSON payloads. Supports OData v4. Enables construction of OData services and clients. Targets .NET Platform Standard 1.1.
-OData .NET library is open source at http://github.com/OData/odata.net. Documentation for the library can be found at https://odata.github.io/odata.net.</description>
+OData .NET library is open source at http://github.com/OData/odata.net. Documentation for the library can be found at https://docs.microsoft.com/en-us/odata/.</description>    

```

Comment:
>https://docs.microsoft.com/en-us/odata/. [](start = 118, length = 40)

it's unfortunate that we don't have a better landing page specific to odatalib (none of the pages below /odatalib provide a good overview.)